independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/Beans.java

```diff
@@ -373,6 +376,13 @@ static BeanInfo createProducerField(FieldInfo producerField, BeanInfo declaringB
         return bean;
     }
 
+    private static AnnotationInstance normalizedNamedQualifier(String defaultedName, AnnotationInstance originalAnnotation) {
+        // Replace @Named("") with @Named("foo")
+        // This is not explicitly defined by the spec but better align with the RI behavior
```

Comment:
`RI`?

Reply:
Reference Implementation. Welcome to Jakarta EE :)

Reply:
LOL